test/mitmproxy/test_websocket.py

```diff
@@ -26,3 +28,18 @@ def test_basic(self):
         assert not m.killed
         m.kill()
         assert m.killed
+
+    def test_text(self):
+        txt = websocket.WebSocketMessage(Opcode.TEXT, True, b"foo")
+        bin = websocket.WebSocketMessage(Opcode.BINARY, True, b"foo")
+
+        assert txt.is_text
+        assert txt.text
```

Comment:
I think you meant to compare the `text` property with something here

Reply:
Or maybe you just wanted to see if it raises, but might as well `== 'foo'` to check the conversion?

Reply:
Yes, exactly.